src/main/java/org/apache/sling/junit/impl/servlet/JUnitServlet.java

```diff
@@ -59,8 +89,10 @@
 
     private volatile ServletProcessor processor;
 
-    protected void activate(final ComponentContext ctx) throws ServletException, NamespaceException {
-        servletPath = getServletPath(ctx);
+    @Activate
+    @Modified
```

Comment:
remove

Reply:
Fixed in the latest changes. Will do another pull request.